.env.production

```diff
@@ -56,18 +56,18 @@ REACT_APP_PATH_REGEX_ENS="/ipfs"
 #REACT_APP_SENTRY_AUTH_TOKEN='<sentry_auth_token>'
 
 # API
-#REACT_APP_API_URL_PROD_MAINNET=https://protocol-mainnet.gnosis.io/api
-#REACT_APP_API_URL_PROD_RINKEBY=https://protocol-rinkeby.gnosis.io/api
-#REACT_APP_API_URL_PROD_XDAI=https://protocol-xdai.gnosis.io/api
-#REACT_APP_API_URL_STAGING_MAINNET=https://protocol-mainnet.dev.gnosisdev.com/api
-#REACT_APP_API_URL_STAGING_RINKEBY=https://protocol-rinkeby.dev.gnosisdev.com/api
-#REACT_APP_API_URL_STAGING_XDAI=https://protocol-xdai.dev.gnosisdev.com/api
+#REACT_APP_API_URL_PROD_MAINNET=https://api.cow.fi/mainnet
+#REACT_APP_API_URL_PROD_RINKEBY=https://api.cow.fi/rinkeby
+#REACT_APP_API_URL_PROD_XDAI=https://api.cow.fi/xdai
+#REACT_APP_API_URL_STAGING_MAINNET=https://barn.api.cow.fi/mainnet
+#REACT_APP_API_URL_STAGING_RINKEBY=https://barn.api.cow.fi/rinkeby
+#REACT_APP_API_URL_STAGING_XDAI=https://barn.api.cow.fi/xdai
```

Comment:
Barn apis are not yet available and we'll release without them for now. Can you please revert them?